engo_glfw.go

```diff
@@ -317,7 +352,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
```

Comment:
I make here the same point. `Engo` is a `lib` and as a developer I want the flexibility to setup my project as I want. This is the issue I was talking about on the other issue. There is no need for the `render` to know about scenes or worlds.

If I want to use another pattern with different kind of approach I can't because I have to set this up with ecs elements.